pyanaconda/ui/gui/spokes/custom_storage.py

```diff
@@ -682,9 +700,10 @@ def _update_all_devices_in_selectors(self):
         for s in self._accordion.all_selectors:
             for new_device in self._storage_playground.devices:
                 if ((s._device.name == new_device.name) or
-                    (getattr(s._device, "req_name", 1) == getattr(new_device, "req_name", 2)) and
-                    s._device.type == new_device.type and
-                    s._device.format.type == new_device.format.type):
+                        (getattr(s._device, "req_name", 1) == getattr(new_device, "req_name",
+                                                                      2)) and
+                        s._device.type == new_device.type and
+                        s._device.format.type == new_device.format.type):
```

Comment:
Could you please change this part to:

```
if not (s._device.name == new_device.name):
    continue
if ((getattr(s._device, "req_name", 1) == getattr(new_device, "req_name", 2)) and
        s._device.type == new_device.type and
        s._device.format.type == new_device.format.type):
    # code snippet
```

Reply:
I have replaced `s._device` and `new_device` with new variables to make this code more readable. I don't want to do more changes at this moment, because the condition doesn't look correct to me.

Reply:
Nice, thanks!